packages/medusa/src/loaders/module-definitions/index.ts

```diff
@@ -40,9 +40,16 @@ export default ({ modules }: ConfigModule) => {
       )
     }
 
+    const moduleDeclaration =
+      typeof moduleConfiguration === "object" ? moduleConfiguration : {}
```

Comment:
todo: null is an object too. should we should check that

Reply:
that's ok, the spread operator will ignore it
```
moduleDeclaration: {
  ...definition.defaultModuleDeclaration,
  ...moduleDeclaration,
},
```